examples/production-cluster-with-metrics.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "database.arangodb.com/v1alpha"
```

Comment:
Can you switch to `database.arangodb.com/v1`? Even, you can do it in all example files

Reply:
Yes sure! Will take care of that!